docs/content/docs/projects/CiSteps.md

```diff
@@ -1,52 +1,68 @@
 ---
-title: CI Gradle Plugin
+title: CI steps plugin
 type: docs
 ---
 
-# CI Gradle plugin
+# CI steps plugin
 
-Для настройки CI под конкретный Gradle модуль используется in-house плагин.
+{{< hint warning>}}
+Plugin wasn't tested outside Avito yet, so expect difficulties, or even blockers.
+However, if you interested, please [contact us]({{< ref "/docs/Contacts.md" >}})!
+{{< /hint >}}
 
-Решает задачу точечной настройки типа и строгости проверок для разных сценариев.
+Plugin creates chains of tasks for CI, encapsulating it under single gradle task.
+
+For example, scenario "Pull Request checks".\
+Typically, you need to create multiple CI builds and wire it into chain.\
+Chain configuration(and possibly parameters) leaks to CI server settings, and you have less control over its changes inside gradle.\
+It makes sense if resources of single machine limited, but if all chain could run effectively
+(e.g. tasks runs in parallel and server has extra resources) on single machine, having all configuration in Gradle has benefits.
```

Comment:
I didn't get the idea of the last two sentences. What do we want to explain? What do we lose without them?

Reply:
My proposal is to show the idea throw [example](https://github.com/avito-tech/avito-android/pull/388/files#r425435432) and illustrate tasks graphically.

Reply:
We can compare two opposite setups: chain inside Gradle and chain inside TeamCity